src/Fixer/Phpdoc/PhpdocAlignFixer.php

```diff
@@ -35,6 +35,7 @@
 {
     private $regex;
     private $regexCommentLine;
+    private $collapse = false;
```

Comment:
The default value is useless as it always will be overridden in `configure`.

Reply:
We can refactor this config a bit. You are welcome [to the discussion](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3453#discussion_r169171531). 😃

Reply:
No much to refactor, it's simply about removing ` = false` part - real setting of default value is done `createConfigurationDefinition` method anyway.